stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -2147,6 +2148,20 @@ impl NakamotoChainState {
             &receipt.header.anchored_header.block_hash()
         );
 
+        let mut tx_receipts = receipt.tx_receipts.clone();
```

Comment:
Why not just modify `receipt.tx_receipts` directly?

Reply:
Good catch, fixed!